state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -426,16 +446,6 @@ pub mod pallet {
 			asset: TargetChainAsset<T, I>,
 			disabled: bool,
 		},
-		EgressScheduled {
```

Comment:
we do use this event in the SDK at the moment. so we will need a bit of time to adjust our code when this is released to a testnet

Reply:
👍 I think this will be released a bit slower than last time due to breaking changes like this. Will stay on testnet for a while before moving across.